scripts/syschdemd.nix

```diff
@@ -0,0 +1,54 @@
+{ 
+  stdenvNoCC, 
+  makeWrapper, 
+  lib, 
+
+  coreutils, 
+  daemonize,
+  glibc,
+  gnugrep,
+  systemd, 
+  util-linux, 
+  
+  defaultUser,
+  defaultUserHome
+}:
```

Comment:
Can we use nixpkgs-fmt instead of alejandra?

Reply:
This is actually formatted manually